frb/halos/hmf.py

```diff
@@ -77,7 +77,13 @@ def frac_in_halos(zvals, Mlow, Mhigh, rmax=1.):
         ratios: ndarray
           rho_halo / rho_m
     """
-
+    # Deal with scalar input
+    if np.isscalar(zvals):
+        zvals = np.array([zvals])
```

Comment:
do this:

`zvals = np.atleast_1d(zvals)`

Reply:
Done.